### PR TITLE
BA-1196: add contributing and code of conduct

### PR DESCRIPTION
**Pull request by deboracosilveira**
Add contributing and code of conduct

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+5. Assign appropriate labels and milestones to the issue, if you have access.
+
+## Requesting Features
+
+If you have an idea for a new feature or enhancement in BaseApp, please follow these steps to request it:
+
+1. Check the [GitHub Issues](https://github.com/silverlogic/baseapp-frontend/issues) to ensure that the feature hasn't already been requested.
+
+2. If the feature hasn't been requested, create a new issue by clicking on the "New Issue" button.
```
Comment (anicioalexandre) [inline CONTRIBUTING.md]: let's make sure people use the right label too, maybe for features requests we could use the [enhancement](https://github.com/silverlogic/baseapp-frontend/labels/enhancement) label

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+We welcome code contributions to BaseApp! To contribute code, please follow these steps:
+
+1. Fork the BaseApp repository to your GitHub account.
+
+2. Clone your forked repository to your local development environment.
+
+3. Create a new branch for your feature or bug fix:
+
+   ```
+   git checkout -b feature-name
```
Comment (anicioalexandre) [inline CONTRIBUTING.md]: if it's not tied to any jira ID we could use `BA-feature-name`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+   ```
+
+4. Make your changes, write tests if applicable, and ensure that the existing tests pass.
+
+5. Commit your changes with a clear and concise commit message:
+
+   ```
+   git commit -m "Add feature: Your feature description"
```
Comment (anicioalexandre) [inline CONTRIBUTING.md]: same here, if not tied to any id we could use "BA: feature description"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+6. Push your changes to your forked repository:
+
+   ```
+   git push origin feature-name
+   ```
+
+7. Open a Pull Request (PR) against the `master` branch of the BaseApp repository.
+
+8. Provide a descriptive PR title and include details about the changes you've made.
```
Comment (anicioalexandre) [inline CONTRIBUTING.md:72]: maybe we could add another step to make sure people read the [versioning](https://github.com/silverlogic/baseapp-frontend/blob/master/README.md?plain=1#L96) section on the readme

**Review by anicioalexandre — APPROVED** [review]
I like it!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+---
+
+## Reporting Bugs
+
+When you encounter a bug or unexpected behavior in BaseApp, please follow these steps to report it:
+
+1. Check the [GitHub Issues](https://github.com/silverlogic/baseapp-frontend/issues) to ensure that the bug hasn't already been reported.
+
+2. If the issue hasn't been reported, create a new issue by clicking on the "New Issue" button.
```
Comment (anicioalexandre) [inline CONTRIBUTING.md]: let's make sure people use the right label too, maybe for bugs requests we could use the [bug](https://github.com/silverlogic/baseapp-frontend/labels/bug) label

**Review by pablogadhi — APPROVED** [review]

**Pull request merged**